src/ray/raylet/worker_pool_test.cc

```diff
@@ -278,7 +278,7 @@ class WorkerPoolMock : public WorkerPool {
                                                                client_call_manager_,
                                                                worker_startup_token);
     std::shared_ptr<WorkerInterface> worker =
```

Comment:
Two things:
- Could you please rename `worker_` to `worker`? google coding style suggests to use underscore only for data members;
- Casting is not necessary here, see my example code: https://onlinegdb.com/3ShpMspUN